tests/5.0/program_control/test_set_and_get_omp_affinity.c

```diff
@@ -0,0 +1,49 @@
+//===--- test_set_and_get_omp_affinity.c ------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// This test checks the modification and retreival of the affinity-format-var 
+// through the use of the omp_set_affinity_format() and omp_get_affinity_format() 
+// routines. The omp_display_affinity() routine is used to display affinity format.
+//
+// Adopted from OpenMP 5.0 Example affinity_display.3.c
+//===------------------------------------------------------------------------===//
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <stddef.h>
+#include <string.h>
+#include <omp.h>
+#include "ompvv.h"
+
+#define FORMAT_STORE 80 
+
+int main () {
+
+   int i, errors, n, thrd_num;
+   size_t nchars;
+   size_t set_nchars;
+   errors = 0;
+    
+   char default_format[FORMAT_STORE];
+   char my_format[] = "host=%20H thrd_num=%0.4n binds_to=%A";
+   char **buffer;
+
+   // Display Default Affinity Format using NULL
+   omp_display_affinity(NULL);
+   printf("\n");
```

Comment:
Convert to info msg.

Reply:
@spophale Decided to remove lines 33 and 34 instead of converting to info_msg.